x-pack/plugins/security_solution/public/overview/components/overview_cti_links/translations.ts

```diff
@@ -53,15 +53,14 @@ export const DANGER_TITLE = i18n.translate(
 export const DANGER_BODY = i18n.translate(
   'xpack.securitySolution.overview.ctiDashboardEnableThreatIntel',
   {
-    defaultMessage:
-      'You need to enable the filebeat threatintel module in order to view data from different sources.',
+    defaultMessage: 'You need to enable threat intel sources in order to view data.',
   }
 );
 
 export const DANGER_BUTTON = i18n.translate(
   'xpack.securitySolution.overview.ctiDashboardDangerPanelButton',
   {
-    defaultMessage: 'Enable Module',
+    defaultMessage: 'Enable sources',
```

Comment:
I think we might want to have a new translation such as `'xpack.securitySolution.overview.ctiDashboardDangerButton'` as not doing so might cause a build error (I believe "update" is not supported once a translation has been released)